assets/charts/components/headlamp/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
```

Comment:
As this is our own chart, I don't think we should use helpers. We can easily set values in templates. We are also not using this in webhook-server.